packages/wallet/src/cip30.ts

```diff
@@ -75,6 +75,7 @@ export type CallbackConfirmation = {
   signTx: (args: SignTxCallbackParams) => Promise<boolean>;
   submitTx: (args: SubmitTxCallbackParams) => Promise<boolean>;
   getCollateral?: GetCollateralCallback;
+  onNoWalletError?: (error: ApiError) => void;
```

Comment:
Does this belong in this PR? :thinking: 
```suggestion
```

Reply:
done in [63c5f161c3dd48eef2a5e3b0225d41f662530e99](https://github.com/input-output-hk/cardano-js-sdk/pull/1192/commits/63c5f161c3dd48eef2a5e3b0225d41f662530e99)